README.md

```diff
@@ -18,7 +18,17 @@ Community Committee meetings will be broadcast via Zoom, will be announced ahead
 ## Current Initiatives, Teams, and Working Groups
 
 ### Initiatives
-Initiatives are projects that the Community Committee and the broader community members are collaborating on to enable Node.js across the ecosystem. Current initiatives are listed in [STRATEGIC-INITIATIVES.md](./STRATEGIC-INITIATIVES.md).
+Initiatives are projects that the Community Committee and the broader community members are collaborating on to enable Node.js across the ecosystem. A complete list of current initiatives is available at [STRATEGIC-INITIATIVES.md](./STRATEGIC-INITIATIVES.md).
+
+If you're looking to get started with participating, the following are fairly straightforward initiatives that could use your help:
+
+| Initiative         | Champion                        | Links                                                            |
+|--------------------|---------------------------------|------------------------------------------------------------------|
+| i18n               | Ben Michel                      | https://github.com/nodejs/i18n                                   |
```

Comment:
it might be better to mention champions by their usernames rather than their names to make it easier to contact them here. in order, that would be: @obensource @dshaw @oe @amiller-gh